apps/web/src/common/vault.ts

```diff
@@ -184,5 +184,18 @@ class Vault {
       }
     });
   }
+
+  static async lockVault() {
+    await db.vault.lock();
+    showToast("success", strings.vaultLockedSuccessfully());
+  }
+
+  static async doesExist() {
+    return db.vault.exists();
+  }
+
+  static isLocked() {
+    return !db.vault.unlocked;
+  }
```

Comment:
This is unnecessary. Use `db.vault.unlocked` directly. No need for this abstraction.